STYLEGUIDE.md

```diff
@@ -8,70 +8,79 @@ the version on the master branch should be followed, as this version should be t
-      1. [Enumerate all cases or have a default case](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#enumerate-all-cases-or-have-a-default-case)
-   1. [Annotations](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#annotations)
-   1. [Comments](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#comments)
-   1. [Modifiers](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#modifiers)
-   1. [Numeric literals](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#numeric-literals)
+      2. [No horizontal alignment](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#no-horizontal-alignment)
+   6. [Optional grouping parentheses: recommended](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#optional-grouping-parentheses-recommended)
+   7. [Enum classes](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#enum-classes)
+      1. [Comment fall-through](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#comment-fall-through)
+      2. [Enumerate all cases or have a default case](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#enumerate-all-cases-or-have-a-default-case)
+   8. [Annotations](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#annotations)
+   9.  [Comments](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#comments)
+      1.  [Do you need the comment?](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#do-you-need-the-comment)
+   10. [Modifiers](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#modifiers)
+   11. [Numeric literals](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#numeric-literals)
       1. [Underscores in numeric literals](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#underscores-in-numeric-literals)
-      1. [Long suffixes](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#long-suffixes)
-   1. [Variable declarations](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#variable-declarations)
-      1. [One variable declaration per line](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#one-variable-declaration-per-line)
-      1. [Declare as close as possible to use](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#declare-as-close-as-possible-to-use)
-   1. [Arrays](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#arrays)
-      1. [No C-style array declarations](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#no-c-style-array-declarations)
-   1. [Indentation](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#indentation)
-      1. [4 Spaces, no tabs](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#4-spaces-no-tabs)
-      1. [Aligning method calls](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#aligning-method-calls)
```

Comment:
This also isn't correctly indented in the table of contents

Reply:
I think there is a problem with the markdown extensions i'm using on vscode because I didn't see these problems while editing there. I am going to check it over again and edit directly on github.